app/controllers/idv/please_call_controller.rb

```diff
@@ -6,8 +6,8 @@ class PleaseCallController < ApplicationController
 
     def show
       analytics.idv_please_call_visited
-      verified_at = current_user.profiles.last.verified_at || Time.zone.today
-      @call_by_date = verified_at + FRAUD_REVIEW_CONTACT_WITHIN_DAYS
+      pending_at = current_user.profiles.last.fraud_review_pending_at || Time.zone.today
```

Comment:
suggestion: we can use `User#fraud_rejection_profile` here instead of `User#profiles.last`

Reply:
Done